src/app/billing-plans/page.tsx

```
@@ -0,0 +1,513 @@
+          <h2 className="text-xl font-bold mb-4">
+            {t("errorTitle", "Error Loading Plans")}
+          </h2>
+          <p className="mb-4">{error}</p>
+          <button
+            onClick={() => {
+              setError("");
+              setIsLoading(true);
+              fetchPlans();
+            }}
+            className="px-4 py-2 bg-purple-600 hover:bg-purple-700 text-white rounded-lg transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)